### ISSUE TITLE
Rubocop >= 1.20.0 breaks with this gem

### ISSUE THREAD

**Issue by pantajoe** [covermymeds/rubocop-thread_safety#42]
Hello 👋🏼 Thanks for your gem!

Unfortunately, when upgrading to rubocop 1.20.0, running `rubocop` breaks when including `rubocop-thread_safety` with this error message:

```
An error occurred while ThreadSafety/MutableClassInstanceVariable cop was inspecting {full filepath}
```

This is due to a [change](https://github.com/rubocop/rubocop/compare/v1.19.1...v1.20.0#diff-13f1a65dc2e836336e800d966db858930c8c395fbbae5d674af415c7819d9e74) to the constant `FROZEN_STRING_LITERAL_TYPES`.

Your code relies on this constant here:
https://github.com/covermymeds/rubocop-thread_safety/blob/9d5fa78e5cc8512f0f2979e4f46fef5642dbe817/lib/rubocop/cop/thread_safety/mutable_class_instance_variable.rb#L181

I believe you can simply delete your own helper method `frozen_string_literal?(node)`  as the module `FrozenStringLiteral` defines this as intended.

**Comment by biinari** [comment]
Thanks @pantajoe for spotting this and for giving enough detail on what's needed to fix it.

**Comment by mikegee** [comment]
version 0.4.3 is released with the fix. Thank you @pantajoe and @biinari!

**Comment by johanlunds** [comment]
Just FYI, I think this broke Rubocop < 1.20.0 or something like that.

We are running Rubocop 1.18.4 and get this error now:

```
An error occurred while ThreadSafety/MutableClassInstanceVariable cop was inspecting /usr/src/app/spec/support/matchers/dry_validation.rb:27:6.
undefined method `frozen_string_literal?' for #<RuboCop::Cop::ThreadSafety::MutableClassInstanceVariable:0x000000400d8ac530>
Did you mean?  frozen_string_literals_enabled?
/usr/local/bundle/gems/rubocop-thread_safety-0.4.3/lib/rubocop/cop/thread_safety/mutable_class_instance_variable.rb:150:in `check'
[....]

Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.18.4 (using Parser 3.0.2.0, rubocop-ast 1.11.0, running on ruby 2.6.8 x86_64-linux-musl)
```

**Comment by johanlunds** [comment]
I will try upgrading Rubocop to see if that fixes it.

**Comment by johanlunds** [comment]
Some minimal example code:

```ruby
module DryValidationMatchers
  CORRECT_VALUE_FOR_TYPE = {
    # ...
  }.freeze

  RSpec::Matchers.define :validate do |field_name|
    match do |form_validation|
      @cases = [] # it's failing here
    end

    private

    def validate_required
      @cases << 'foobar'
    end
  end
end
```

**Comment by biinari** [comment]
My mistake, I hadn't checked how far back that method was defined by rubocop. This should fix it: #44

**Comment by mikegee** [comment]
0.4.4 is released now. thanks @johanlunds and @biinari
